ArduCopter/Copter.h

```diff
@@ -471,6 +471,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
     // 3D Location vectors
     // Current location of the copter (altitude is relative to home)
     Location_Class current_loc;
+    Location ekf_origin;
```

Comment:
I'm not sure it's a great idea to keep a copy of this around.  I know it's a bit more code and checking.. but better to just get it straight from the ekf each time.